pkg/server/authorizedentries/cache.go

```diff
@@ -164,6 +192,26 @@ func (c *Cache) appendDescendents(records []entryRecord, parentID string, parent
 	return records
 }
 
+func (c *Cache) findDescendents(foundEntries map[string]*types.Entry, parentID string, requestedEntries map[string]struct{}, parentSeen stringSet) {
```

Comment:
I think it should be _findDescendants_?

Also, I wonder if the name might be confusing since it's not immediately obvious that it mutates the foundEntries map by adding descendants. Would addDescendants be a more appropriate name?